pkg/routeagent/controllers/route/route.go

```diff
@@ -597,6 +601,15 @@ func (r *Controller) cleanRoutes() {
 	}
 }
 
+func (r *Controller) cleanStrongswanRoutingTable() {
+	cmd := exec.Command("/sbin/ip", "r", "flush", "table", "220")
```

Comment:
This API is tied to Strongswan implementation and now that there are some efforts in adding other drivers, shall we add a comment to revisit this API?

Reply:
True, let me extend L467 note to the functions.

I believe this should not be altogether here. Also, in cases where the route agent won't live on every node this won't work.

We need to find a better place, the alternative would be having a separate daemonset to run on nodes, but that sounds expensive... we need to keep thinking about it.

Reply:
I updated the comments, let me know if it looks ok :)

Reply:
Thanks for adding the comments.